### DISCUSSION TITLE
✨ Daggerverse: Index of a user's modules

### DISCUSSION THREAD

**Discussion by kpenfound** [dagger/dagger#12154] [Ideas]
### What are you trying to do?

> it would be great to view all of a user's modules from something like https://daggerverse.dev/mod/github.com/levlaz/daggerverse  (i.e. removing the module name from the URL to get to an index, similar to how you can do this on github)

Further, a github user's path would be good too if the user doesn't use a daggerverse repo. So `https://daggerverse.dev/mod/github.com/levlaz`

### How are you currently working around this?

Search for the username in daggerverse
https://daggerverse.dev/search?q=levlaz

**Comment by jpadams** [comment]
Since a given author might have modules in many git repos in many git servers under different user/org names, it's an interesting question of "who is a user" (since we don't have a Daggerverse login today).
You can make a search today that might accidentally work like:
> lev1 OR lev2
real example: https://daggerverse.dev/search?q=jpadams+OR+levlaz

It could make sense to allow search by git username/orgname through something like:
> git-org:levlaz
